pkg/session/session.go

```diff
@@ -4296,6 +4297,26 @@ func (s *session) NewStmtIndexUsageCollector() *indexusage.StmtIndexUsageCollect
 	return indexusage.NewStmtIndexUsageCollector(s.idxUsageCollector)
 }
 
+// isPipelinedDML returns the current statement can be executed as a pipelined DML.
+func (s *session) isPipelinedDML() bool {
+	if !s.sessionVars.BulkDMLEnabled {
+		return false
+	}
+	stmtCtx := s.sessionVars.StmtCtx
+	if stmtCtx == nil {
+		return false
+	}
+	if !stmtCtx.InInsertStmt && !stmtCtx.InDeleteStmt && !stmtCtx.InUpdateStmt {
```

Comment:
Just a question, how about `LOAD DATA`? We may not support it but it's good to clarify. Because in release notes we mentioned "any DML"

Reply:
Just claim we donot support `LOAD DATA` yet(maybe in the future)?

Reply:
`LOAD DATA` is a main entry for the usages of large tranctions, we need to clarify with PM if we decide not to support it by now.